terminusdb_client/woqlquery/woql_query.py

```diff
@@ -3329,3 +3329,16 @@ def doctype(self, user_type, graph=None, label=None, description=None):
         if isinstance(description, str) and description:
             result_obj = result_obj.description(description)
         return result_obj
+
+    def vars(self, varray):
+        """Generate variables to be used in WOQLQueries
+        Parameters
+        ----------
+        varray : list
+            list of strings
+        Returns
+        -------
+        list
+            List of strings prefixed with "v:"
+        """
+        return list(map(lambda var: "v:" + var, varray))
```

Comment:
One small thing, we want to have it return a bunch of string instead of a list of string.
For example.
var_a, var_b = WOQLQuery().vars("var1", "var2")

Hints: You can use * to unpack lists.

Reply:
Got it